usr.bin/elfctl/elfctl.c

```diff
@@ -68,6 +68,8 @@ static struct ControlFeatures featurelist[] = {
 	{ "nostackgap",	NT_FREEBSD_FCTL_STKGAP_DISABLE, "Disable stack gap" },
 	{ "wxneeded",	NT_FREEBSD_FCTL_WXNEEDED, "Requires W+X mappings" },
 	{ "la48",	NT_FREEBSD_FCTL_LA48, "amd64: Limit user VA to 48bit" },
+	{ "cheri_revoke", NT_FREEBSD_FCTL_CHERI_REVOKE_ENABLE, "Enable CHERI revocation" },
+	{ "nocheri_revoke", NT_FREEBSD_FCTL_CHERI_REVOKE_DISABLE, "Disable CHERI revocation" },
```

Comment:
Hm, underscores are a bit out of place with the other names, but this would be harder to read without them...

Reply:
Could do CAPREVOKE instead.

Reply:
I convinced myself the control words should be without underscores and have updated things accordingly